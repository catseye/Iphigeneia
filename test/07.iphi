(* Test 'while ... do ...' *)

var a in var b in var c in
begin
    a := 10;
    b := 1;
    c := 2;
    while a > 0 do
        begin
	    b := b * c;
	    c := c + b;
	    a := a - 1
	end
end
