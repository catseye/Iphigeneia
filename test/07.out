a=0
c=140982598893793678070294688422804665931354981644880911847733136248186424030732278900819020480668973702640170212905160639132296847654374706155245147715674612235227680384069415566749494180212370357849936526549755341591854042821940420766722160615645816921368300
b=140982598893793678070294688422804665931354981644880911847733136248186424030732278900819020480668973702640170212905160639132296847278898210361175931159590631877400396153764977561991761037132722898953457959352992281368361865140291306311370294857131871923863552
